submissions/shawn-sean-anthony/scripts/article.js

```diff
@@ -0,0 +1,60 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+
+
+  $newArticle.find('article.template').attr('data-category', this.category);
+  $newArticle.find('h1').text(this.title);
+  $newArticle.find('address a').attr('href', this.authorUrl);
+  $newArticle.find('address a').text(this.author);
+  $newArticle.find('time[pubdate]').attr('datetime',this.publishedOn);
+  $newArticle.find('.article-body').html(this.body);
```

Comment:
This is great. As you can see, the .find() method is rather expressive, in that we can target any matching elements within our $newArticle, and apply some setters accordingly.